sig-federation/ONCALL.md

```diff
@@ -0,0 +1,62 @@
+# Overview
+
+We have an oncall rotation in the SIG. The role description is as follows:
+
+* Ensure that the testgrid (https://k8s-testgrid.appspot.com/sig-federation) is green. This person will be the point of contact if testgrid turns red. Will identify the problem and fix it (most common scenarios: find culprit PR and revert it or free quota by deleting leaked resources).
+Will also report most common failure scenarios and suggest improvements. Its up to the sig or individuals to prioritize and take up those tasks.
+
+Oncall playbook: https://github.com/kubernetes/community/blob/master/contributors/devel/on-call-federation-build-cop.md
+
+# Joining the rotation
+
+Add your name at the end of the current rotation schedule.
```

Comment:
Are there requirements to join? Permissions? Do you have to be a googler? I'm guessing these are important pieces for any on-call rotation.

Reply:
Thanks for pointing that out. Added information to the doc. PTAL